client/clientBean.go

```diff
@@ -55,6 +55,7 @@ type (
 		GetFrontendClient() frontend.Client
 		GetRemoteAdminClient(cluster string) admin.Client
 		GetRemoteFrontendClient(cluster string) frontend.Client
+		SetRemoteFrontendClient(cluster string, client frontend.Client)
```

Comment:
not sure this is a good idea, can you create a dummy client bean use that?

Reply:
I don't have a better idea to inject this mock in an integration test as the test is using service.New. I will add the setter on all variables.